examples/project/embedded/LeanCode.Contracts.pb

```diff
@@ -0,0 +1,6 @@
+
+embedded
```

Comment:
Shouldn't this be in LFS? I'm having a hard time updating the submodule in the dart generator. It complains about git LFS. Not sure if I am doing something wrong or this was committed incorrectly @Saancreed @jakubfijalkowski

Reply:
I'm indifferent. This particular file is just 42 bytes long, that's even smaller than the `DTO.cs` next to it :sweat_smile:

Well, it does contain few unprintable characters but I wouldn't call this a good reason to migrate it to _large_ file storage.

Reply:
It is 42 bytes, so let's just migrate it to LFS as a normal commit.